docs/bad_dump.md

```diff
@@ -0,0 +1,12 @@
+1. `ssh deploy@bibdata-alma-worker1`
+1. `cd /opt/marc_liberation/current`
+1. `RAILS_ENV=production bundle exec rails c`
+1. Find the IndexManager instance that uses the solr production collection. Currenlty is:`catalog-alma-production`
+`index_manager_current = IndexManager.all.first`
+1. `index_manager_current.solr_collection` => `http://lib-solr8-prod.princeton.edu:8983/solr/catalog-alma-production`
```

Comment:
@christinach why is this a step?  What is it doing?

Reply:
its not a step. You make sure that you are checking the correct index manager instance by checking the solr_collection. I'm sorry I added it as a step.

Reply:
there are two instances one has as a solr_collection: catalog-alma-production-rebuild but it is not in production.

Reply:
I will update it